aodndata/srs/srs_ghrsst.py

```diff
@@ -10,20 +10,45 @@
 L3S_L3C_FILE_PATTERN = re.compile(r"""
                                 (?P<nc_time_cov_start>[0-9]{14})-ABOM-
                                 (?P<product_type>L3S|L3C)_.*-AVHRR
-                                (?P<sat_number>.*)_D-
+                                (?P<sat_value>.*)_D-
                                 (?P<temporal_extent>1d|3d|6d|14d|1m)_
                                 (?P<day_time>day|night|dn).*\.nc$
                                 """, re.VERBOSE)
 
+L3S_MULTISENSOR_FILE_PATTERN = re.compile(r"""
+                                (?P<nc_time_cov_start>[0-9]{14})-ABOM-
+                                (?P<product_type>L3S)_.*-MultiSensor-
+                                (?P<temporal_extent>1d|3d|6d|14d|1m)_
+                                (?P<day_time>day|night|dn)
+                                (?P<optional>_Southern|)
+                                \.nc$
+                                """, re.VERBOSE)
+
 L3U_FILE_PATTERN = re.compile(r"""
                                 (?P<nc_time_cov_start>[0-9]{14})-ABOM-
                                 (?P<product_type>L3U)_.*-AVHRR
-                                (?P<sat_number>[0-9].*)_D-
+                                (?P<sat_value>[0-9].*)_D-
                                 (?P<pass_direction>Asc|Des)
                                 (?P<optional>_Southern|)
                                 \.nc$
                                 """, re.VERBOSE)
 
+L3U_VIRRS_FILE_PATTERN = re.compile(r"""
```

Comment:
There is a difference in the file pattern : is it VIRRS or VIIRS?

Reply:
thanks. will correct. it's viirs

Reply:
@bpasquer done !